src/common/users.ts

```diff
@@ -636,3 +639,65 @@ export const bskySocialUrlMatch =
   /^(?:(?:https:\/\/)?(?:www\.)?bsky\.app\/profile\/)?(?<value>[\w.-]+)(?:\/.*)?$/;
 
 export const portfolioLimit = 500;
+
+const MIN_WORK_EXPERIENCE = 1; // Minimum number of work experiences required for profile completion
+export const isProfileCompleteById = async (
+  con: DataSource,
+  userId: User['id'],
+) => {
+  try {
+    const [user, experiencesCount] = await queryReadReplica(
```

Comment:
From when does the FE call this, because it's RR it might not be instant available.